src/ggrc/assets/javascripts/models/cacheable.js

```
@@ -1121,12 +1121,20 @@
         } else if (val && typeof val.save === 'function') {
           serial[name] = val.stub().serialize();
         } else if (typeof val === 'object' && val !== null && val.length) {
-          serial[name] = can.map(val, function (v) {
-            var isModel = v && typeof v.save === 'function';
-            return isModel ?
-            v.stub().serialize() :
-            v.serialize ? v.serialize() : v;
-          });
+          serial[name] = Array.prototype.map
+            .call(val, function (v) {
+              var isModel = v && typeof v.save === 'function';
+              if (!v) {
+                return false;
+              }
+              return isModel ?
+              v.stub().serialize() :
+              v.serialize ? v.serialize() : v;
+            })
+            // Filter falsy values if presented
+            .filter(function (val) {
+              return !!val;
```

Can you revert this change? We don't want issues like the one that this PR solves silently ignored.